go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/warptools/warpforge
 
 //replace github.com/ipld/go-ipld-prime => /home/eric/git/go-ipld-prime
+//replace github.com/ipld/go-ipld-prime => ../go-ipld-prime
```

Comment:
We should probably remove both this and the old one, which shouldn't have been here either :)